src/utils/configs.ts

```diff
@@ -41,3 +41,12 @@ export const regions = [
     { name: "US East (N. Virginia)", value: "us-east-1" },
     { name: "Europe (Frankfurt)", value: "eu-central-1" },
 ];
+
+export const databaseRegions = [
+    { name: "US East (N. Virginia)", value: "us-east-1" },
+    { name: "US East (Ohio)", value: "us-east-2" },
+    { name: "US West (Oregon)", value: "us-west-2" },
+    { name: "Asia Pacific (Singapore)", value: "ap-southeast-1" },
+    { name: "Asia Pacific (Sydney)", value: "ap-southeast-2" },
+    { name: "Europe (Frankfurt)", value: "eu-central-1" },
```

Comment:
The backend won't accept these regions for database creation. You either:
1. add `aws-` in front of each region here
2. append it when sending the request so the user does not see the `aws-` thing
3. modify the backend to receive these types of regions and append the `aws-` thing when forwarding the request to the neon db service.

Reply:
Nice catch, I have chosen 2 and implemented it in a subsequent PR on supporting databases in YAML.
https://github.com/Genez-io/genezio/pull/1277